### PR TITLE
CRS-280 Create data model for calcultion-request and response + create JPA classes and fix int tests

### PR DESCRIPTION
**Pull request by yasinmustafa**
Also:
- Changed alerts channel to farsight-alerts
- Added a upgrade command into Dockerfile as advised by devops

TODO
- Populate calculation requests and results to the new tables
- add tests to check the population

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/entity/CalculationOutcome.kt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table
+@TypeDefs(
+  TypeDef(name = "jsonb", typeClass = JsonBinaryType::class)
+)
```
Comment (timharrison-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/entity/CalculationOutcome.kt:18]: As there is no JSONB types in this entity it probably doesn't need the @Typedefs?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/entity/CalculationRequest.kt [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+@Table
+@TypeDefs(
+  TypeDef(name = "json", typeClass = JsonType::class)
+)
+data class CalculationRequest(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long = -1,
+  val calculationReference: UUID = UUID.randomUUID(),
+  val prisonerId: String = "",
```
Comment (timharrison-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/entity/CalculationRequest.kt:28]: Is null better than empty strings "" for default values? Or are the columns not nullable?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/entity/CalculationOutcome.kt (hunk 1 above)
Comment (yasinmustafa) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/entity/CalculationOutcome.kt:18]: good spot - copy/paste error

File: src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/entity/CalculationRequest.kt (hunk 2 above)
Comment (yasinmustafa) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/entity/CalculationRequest.kt:28]: They are actually not nullable.. will annotate with @NotNull

**Review by timharrison-moj — DISMISSED** [review]
LGTM

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**